src/Operation/FindOneAndUpdate.php

```diff
@@ -149,14 +143,11 @@ public function __construct($databaseName, $collectionName, $filter, $update, ar
      * @see Executable::execute()
      * @param Server $server
      * @return array|object|null
-     * @throws UnsupportedException if array filters or collation or write concern is used and unsupported
+     * @throws UnsupportedException if collation or write concern is used and unsupported
```

Comment:
I think this can still mention array filters, since it throws anything that `FindAndModify::execute()` throws. We can say "array filters, collation, or write concern..." to condense the line.